README.md

```diff
@@ -1,12 +1,45 @@
-My thoughts on the takeaway challenge
+## Takeaway Challenge ##
+==================
+
+## How to Use
+
+```shell
+irb -r './lib/order.rb'
+```
+Create a new order, then choose any of the following methods:
+view_menu --> displays the menu
+select(dish, quantity) --> eg ("dishname", 3)
+place_order --> this submits your order
+
+eg.
+```irb
+my_order = Order.new
+Welcome! Please use view_menu to see the menu.
+To add a dish to your order, use select(dish, quantity)
+To place your order, use place_order.
+2.6.5 :002 > my_order.view_menu
+Pizza: £6.50
+Spaghetti: £5.75
+Salad: £5.25 => nil
+2.6.5 :003 > my_order.select("pizza",3)
+=> "3 pizza(s) added to your order."
+```
+
+
+## My thoughts on the takeaway challenge
```

Comment:
This is also really useful, really nice to get a sense of how you feel about everything and an insight into your thinking